google/cloud/spanner/internal/connection_impl.h

```diff
@@ -59,11 +62,18 @@ class ConnectionImpl : public Connection {
   friend class SessionHolder;
   StatusOr<SessionHolder> GetSession();
 
+  /// Implementation details for Read.
+  StatusOr<ResultSet> Read(google::spanner::v1::TransactionSelector& s,
+                           ReadParams rp);
+
+  /// Implementation details for ExecuteSql
+  StatusOr<ResultSet> ExecuteSql(google::spanner::v1::TransactionSelector& s,
+                                 std::int64_t seqno, ExecuteSqlParams esp);
+
   /// Implementation details for Commit.
   StatusOr<CommitResult> Commit(google::spanner::v1::TransactionSelector& s,
-                                std::vector<Mutation> mutations);
+                                CommitParams cp);
```

Comment:
This change seems unrelated, other than it is more consistent.  Maybe split to a separate PR?